code/controllers/subsystem/airflow.dm

```diff
@@ -12,7 +12,7 @@
 SUBSYSTEM_DEF(airflow)
 	name = "Airflow"
 	wait = 2
-	subsystem_flags = SS_NO_INIT
+	subsystem_flags = SS_NO_INIT | SS_BACKGROUND
```

Comment:
This is probably not a good idea given that background can be totally starved of CPU time during things like explosions or ZAS events.